src/megaapi_impl.cpp

```diff
@@ -22318,15 +22318,15 @@ int MegaPricingPrivate::getProLevel(int productIndex)
 
 unsigned int MegaPricingPrivate::getGBStorage(int productIndex)
 {
-    if((unsigned)productIndex < gbStorage.size())
+    if((unsigned)productIndex < gbStorage.size() && !type[productIndex])
```

Comment:
Since we will change the variables `gbStorage`/`gbBandwidth` to signed integers, we don't need the `type` check anymore.